spec/lib/configuration_spec.rb

```diff
@@ -1,13 +1,8 @@
+require 'spec_helper'
+
 RSpec.describe Qa::Configuration do
   subject { described_class.new }
 
-  it { is_expected.to respond_to(:cors_headers?) }
-  it { is_expected.to respond_to(:enable_cors_headers) }
-  it { is_expected.to respond_to(:disable_cors_headers) }
-  it { is_expected.to respond_to(:authorized_reload_token=) }
-  it { is_expected.to respond_to(:authorized_reload_token) }
-  it { is_expected.to respond_to(:valid_authority_reload_token?) }
```

Comment:
These are redundant since each of these are tested individually later in the spec.